cellarium/cas/data_preparation/callbacks.py

```diff
@@ -8,13 +8,14 @@
 TOTAL_MRNA_UMIS_COLUMN_NAME = "total_mrna_umis"
 
 
-def calculate_total_mrna_umis(adata: "anndata.AnnData") -> None:
+def calculate_total_mrna_umis(adata: "anndata.AnnData", count_matrix_name: str) -> None:
     """
     Calculate the total mRNA UMIs (Unique Molecular Identifiers) for each observation in the AnnData object and add them
     as a new column in the ``.obs`` attribute. It is recommended to use this callback before data sanitization to
     calculate all expressed genes.
 
     :param adata: The annotated data matrix from which to calculate the total mRNA UMIs.
+    :param count_matrix_name: Where to obtain a feature expression count matrix from. Choice of: 'X', 'raw.X'
```

Comment:
This should probably be an enum if the choices are limited

Reply:
I agree with this